hydrolib/core/io/ini/util.py

```diff
@@ -462,3 +462,28 @@ def is_valid_coordinates_with_num_coordinates_specification() -> bool:
         raise ValueError(error)
 
     return root_validator(allow_reuse=True)(validate_location_specification)
+
+
+def get_key_renaming_root_validator(keys_to_rename: Dict[str, List[str]]):
+    """
+    Gets a root validator that renames the provided keys to support backwards compatibility.
+
+    Args:
+            keys_to_rename Dict[str, List[str]]: Dictionary of keys and a list of old keys that
+            should be converted to the current key.
+    """
+
+    def rename_keys(cls, values: Dict) -> Dict:
+        for current_keyword, old_keywords in keys_to_rename.items():
+            if values.get(current_keyword) is not None:
+                continue
+
+            for old_keyword in old_keywords:
+                if values.get(old_keyword) is not None:
+                    values[current_keyword] = values.get(old_keyword)
```

Comment:
```suggestion
                if (value := values.get(old_keyword)) is not None:
                    values[current_keyword] = value
```

You can use the walrus operator for this (maybe a more specific name, but I couldn't come up with anything :))

Reply:
Cool, I didn't know. I have update the code.